src/react/locations/LocationDetails/LocationDetailsAwsCustom.tsx

```diff
@@ -152,25 +153,46 @@ export default function LocationDetailsAwsCustom({
         />
 
         {!isRingS3Reseller ? (
-          <FormGroup
-            content={
-              //@ts-expect-error fix this when you are working on it
-              <Input
-                name="endpoint"
-                type="text"
-                value={formState.endpoint}
-                onChange={onFormItemChange}
-                autoComplete="off"
-                placeholder="https://hosted-s3-server.internal.example.com:4443"
-              />
-            }
-            label="Endpoint"
-            id="endpoint"
-            required
-            labelHelpTooltip="Endpoint to reach the S3 server, including scheme and port. The
+          <>
+            <FormGroup
+              content={
+                <Input
+                  name="endpoint"
+                  id="endpoint"
+                  type="text"
+                  value={formState.endpoint}
+                  onChange={onFormItemChange}
+                  autoComplete="off"
+                  placeholder="https://hosted-s3-server.internal.example.com:4443"
+                />
+              }
+              label="Endpoint"
+              id="endpoint"
+              required
+              labelHelpTooltip="Endpoint to reach the S3 server, including scheme and port. The
         buckets will have a path-style access."
-            helpErrorPosition="bottom"
-          />
+              helpErrorPosition="bottom"
+            />
+
+            <Stack
+              direction="vertical"
+              style={{ width: 'calc(20.5rem + 2rem + 180px)' }}
+            >
+              <Text color="textPrimary">
+                When using an HTTPS endpoint, you must add the certificate
+                provided by its TLS provider to the truststore. You can check if
+                the certificate is already present in the truststore by clicking
+                the button below, and add it if it is missing.
+              </Text>
+              <Link
+                href="/data/truststore"
+                target="_blank"
+                style={{ alignSelf: 'flex-end' }}
+              >
+                Open trust store <Icon name="External-link" />
+              </Link>
```

Comment:
Is it possible to add some test in the `LocationDetailsAwsCustom.test.tsx`  - even it's legacy test :(